app/dns_utils.py

```diff
@@ -1,20 +1,19 @@
+from app import config
 from typing import Optional, List, Tuple
 
 import dns.resolver
 
 
 def _get_dns_resolver():
     my_resolver = dns.resolver.Resolver()
-
-    # 1.1.1.1 is CloudFlare's public DNS server
-    my_resolver.nameservers = ["1.1.1.1"]
+    my_resolver.nameservers = config.NAMESERVERS
 
     return my_resolver
 
 
 def get_ns(hostname) -> [str]:
     try:
-        answers = _get_dns_resolver().resolve(hostname, "NS")
+        answers = _get_dns_resolver().resolve(hostname, "NS", search=True)
```

Comment:
do you know what this `search` parameter does?

Reply:
For some reason, my branch didn't have this commit: https://github.com/simple-login/app/commit/936d90a5f51489263185f68bca57003802b1ced8 and when replacing the `query()` call with the `resolve()` call, I read the docs about `query()` calling `resolve` and setting `search=True`, so I wanted to retain the same behaviour as the old method call.
When rebasing it, the aforementioned commit ended up into my branch and the parameter got added too. I can remove it if you want.

Reply:
For some reason, my branch didn't have this commit: https://github.com/simple-login/app/commit/936d90a5f51489263185f68bca57003802b1ced8 and when replacing the `query()` call with the `resolve()` call, I read the docs about `query()` calling `resolve` and setting `search=True`, so I wanted to retain the same behaviour as the old method call.
When rebasing it, the aforementioned commit ended up into my branch and the parameter got added too. I can remove it if you want.

Reply:
Oh that's indeed a good catch, it's better to keep the same behavior of `query` by setting `search=True`, thanks for taking a look!